elements/rh-pagination/docs/20-guidelines.md

```diff
@@ -102,13 +102,13 @@ Truncation does not apply to the Compact size because it does not display page n
 
 Both pagination sizes and orientations are horizontally centered below content by default.
 
-![Image of full size pagination with the page input field on the right]({{ '../pagination-alignment-pif-right-a.png' | url }})
+![Image of full size pagination with the page input field on the right]({{ '../pagination-alignment-pif-right-a.png' | url }}){style="--inline-img-max-width:1000px;"}
```

Comment:
We should probably use example shortcode instead of these css props

Reply:
Done. I switched all of the inline images to the example shortcode with a palette of none.